charts/perses/templates/config.yaml

```diff
@@ -12,6 +12,12 @@ data:
   config.yaml: |-
     security:
       readonly: {{ .Values.config.security.readOnly }}
+      {{- if and .Values.config.security.encryptionKeyFile .Values.config.security.encryptionKey }}
+      encryption_key_file: {{ printf "%s/key" (.Values.config.security.encryptionKeyFile | trimSuffix "/") }}
+      {{- end }}
+      {{- if and (not .Values.config.security.encryptionKeyFile) .Values.config.security.encryptionKey }}
```

Comment:
I might be wrong, but from my understanding if user provides a encryption key file he must to provide a encryption key.

Is that right @Nexucis ?

Reply:
That's my understanding as well. But if an encryption key file is provided, and the file is mount on the pod, is there any reason to also add the key in the configuration file, config.yaml?

Reply:
yeah `encryptionKey` and `encryptionKeyFile` are mutually exclusive but they are not mandatory.
If you don't provide it, there is one hardcoded that will be used.